src/python/pants/build_graph/address_test.py

```diff
@@ -100,8 +107,12 @@ def test_address_input_parse_bad_path_component(spec: str) -> None:
     "spec,expected",
     [
         ("a:", "non-empty target name"),
-        ("//:", "non-empty target name"),
-        ("//:@t", "non-empty target name"),
```

Comment:
Why remove these?

Reply:
No real reason... just that knowing the underlying implementation, these tests were redundant.